src/nvim/terminal.c

```diff
@@ -201,7 +201,7 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
-  vterm_state_set_bold_highbright(state, true);
+  vterm_state_set_bold_highbright(state, false);
```

Comment:
I don't think keeping it is needed.

If the default is changed, it will most likely be mentioned in the release notes, which are generally very thorough given how few commits I've noticed between releases.